README.md

```diff
@@ -171,630 +171,476 @@ curl https://raw.githubusercontent.com/grafana/helm-charts/main/charts/loki-stac
-Issuer.yml :
-```C
+apiVersion: v1
+kind: Service
+metadata:
+  name: quote
+  namespace: ambassador
+spec:
+  ports:
+  - name: http
+    port: 80
+    targetPort: 8080
+  selector:
+    app: quote
 ---
-
-apiVersion: cert-manager.io/v1
-
-kind: ClusterIssuer
-
+apiVersion: v1
+kind: Service
 metadata:
-
-name: letsencrypt-prod
-
+  name: echo
+  namespace: ambassador
 spec:
+  ports:
+  - name: http
+    port: 80
+    targetPort: 8080
+  selector:
+    app: echo
+```
 
-acme:
+As the last configuration step, create the mapping for Ambassador.
 
-email: email@sample.com
+#### Configure Mapping for each Hosts
 
-server: https://acme-v02.api.letsencrypt.org/directory
+Ambassador Edge Stack is designed around a [declarative, self-service management model](https://www.getambassador.io/docs/edge-stack/latest/topics/concepts/gitops-continuous-delivery) It means that you can manage the edge with Ambassador Edge Stack is the `Mapping` resource.
+> More info : https://www.getambassador.io/docs/edge-stack/1.13/topics/using/intro-mappings/
 
-privateKeySecretRef:
+This Mapping is managing Edge Stack to route all traffic inbound to the /backend/ path to the quote & also /echo/ to the echo service.
+**Details**
 
-name: letsencrypt-prod
+* name:	It's a string identifying the Mapping (e.g. in diagnostics)
+* prefix:	It's the URL prefix identifying your resource.
+* service: It's the name of the service handling the resource; must include the namespace (e.g. myservice.othernamespace) if the service is in a different namespace than Ambassador Edge Stack.
 
-solvers:
+**Mapping.yml**:
+Creating 2 independent mappings 
 
-- dns01:
+- echo services accessible to quote.mandrakee.xyz AND and quote service accessible to echo.mandrakee.xyz
+  
 
-digitalocean:
+```
+~kapp mapping.yml
+---
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: quote-backend
+  namespace: ambassador
+spec:
+  prefix: /backend/
+  host: quote.mandrakee.xyz
+  service: quote
+ 
+---
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: echo-backend
+  namespace: ambassador
+spec:
+  prefix: /echo/
+  host: echo.mandrakee.xyz
+  service: echo
+  ```
 
-tokenSecretRef:
+### Enabling Proxy Protocol
+There are lot of configuration options available to you when running Ambassador Edge Stack but we will focus on automatic certificate management in the Ambassador Edge Stack in this part. After deploying the Service above  and manually enabling the proxy protocol you have to need to deploy the following Ambassador Module to manage Ambassador Edge Stack to utilize the proxy protocol and then restart Ambassador Edge Stack for the configuration to take effect.
 
-name: digitalocean-dns
+Also you can create load balancer by using configuration like below
```

Comment:
You already added an LB in the beginning. When you installed AES, a DO LB was automatically created.

What we're trying to do is to configure proxy protocol on that DO LB.
https://docs.digitalocean.com/products/kubernetes/how-to/configure-load-balancers/#proxy-protocol

Then we should configure AES Proxy protocol.
 config:
     use_proxy_proto: true

That's it.

Reply:
done